SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1062,6 +1122,82 @@ void USpatialSender::RetireEntity(const Worker_EntityId EntityId)
 	}
 }
 
+void USpatialSender::CreateTombstoneEntity(AActor* Actor)
+{
+	check(Actor->IsNetStartupActor());
```

Comment:
Thoughts on moving this functionality into the `EntityFactory` ?

Reply:
I think that was merged after I wrote the majority of this and I didn't realise it was there. Will move it all over.